src/routers/middleware_router.rs

```diff
@@ -25,19 +27,21 @@ impl MiddlewareRouter {
     }
 
     #[inline]
-    fn get_relevant_map(&self, route: &str) -> Option<&RwLock<Node<(PyFunction, u8)>>> {
+    fn get_relevant_map(
+        &self,
+        route: MiddlewareRouteType,
```

Comment:
just one last nit: `MiddlewareRoute` would make more sense than `MiddlewareRouteType`.

Reason:

If you ask the type of route then it should be `MiddlewareRoute` and not type = `MiddlewareRouteType.`

Reply:
Sure, updated this also.